scripts/Drill.py

```diff
@@ -24,6 +24,7 @@
         app, within_mantid = get_qapplication()
         if 'drillInterface' not in globals():
             drillInterface = DrillView()
+        drillInterface.setWindowBehaviour()
```

Comment:
Do we need this extra call to setWindowBehavior, have the flags and parent not already been set in the DrillView constructor?

Reply:
When you close DRILL I think it just hides the window, which means when you open it again it is using the same presenter as before. If the user closes DRILL, changes the window behaviour, and then opens DRILL, the window flags don't match what they should be according to the settings (unless we do something like this)

Reply:
Ah I see, makes sense then